scenes/inventory.py

```diff
@@ -16,7 +16,7 @@
 class InventoryScene(BaseScene):
```

Comment:
My understanding of a scene/window is a scene is a screen and windows are parts of it.
The Inventory takes the entire screen so there are no uses for keeping the game scene at all.
If we are to change this then we must handle windows stopping other windows from rendering/handling UI and the windows that can coexist.